test/Integration/IntegrationTestBase.cs

```diff
@@ -189,7 +188,19 @@ protected void StartFunctionHost(string functionName, bool useTestFolder = false
             this.FunctionHost.BeginOutputReadLine();
             this.FunctionHost.BeginErrorReadLine();
 
-            Thread.Sleep(10000);     // This is just to give some time to func host to start, maybe there's a better way to do this (check if port's open?)
+            var taskCompletionSource = new TaskCompletionSource<bool>();
```

Comment:
We don't actually need the bool, but the framework version we target doesn't have a non-generic TaskCompletionSource so we have to specify something.